web/db/actions/Account.ts

```diff
@@ -47,10 +47,19 @@ async function addAccount(inputData: IAccount, session?: ClientSession) {
   }
 }
 
+async function findAll(session?: ClientSession) {
```

Comment:
Specify return type for this function to be `Promise<HydratedDocument<IAccount>[]>`